string.md

```diff
@@ -7,19 +7,22 @@ toc: true
 
 The `STRING` [data type](data-types.html) stores a string of characters.
 
-In CockroachDB, the following are aliases for `STRING` and `STRING(n)`: 
+In CockroachDB, the following are aliases for `STRING`: 
 
 - `CHARACTER`
-- `CHARACTER(n)`
 - `CHAR` 
-- `CHAR(n)` 
 - `VARCHAR`
-- `VARCHAR(n)` 
 - `TEXT`
 
+And the following are aliases for `STRING(n)`:
+
+- `CHARACTER(n)`
+- `CHAR(n)` 
+- `VARCHAR(n)` 
+
 ## Length
```

Comment:
CockroachDB also does not blank-pad non-VAR CHAR values to the full limit length as in traditional SQL.